File: java/Visitor/src/Order.java
package Exercise;

import java.util.ArrayList;

public class Order implements _______________ { // <<5

	String name;

	private ArrayList<String> items = new ArrayList<String>();

	public Order(String name) {
		this.name = name;
	}

	public void accept(Visitor visitor) {
		visitor._______________(this); // <<6
	}

	public void addItem(String item) {
		items.add(item);
	}

}

File: ruby/Visitor/order.rb
require_relative 'visitable'
require_relative 'visitor'

class Order < _______________ # <<5
  attr_accessor :name
  def initialize(name)
    @name = name
    @items = Array.new
  end

  def accept(visitor)
    visitor._______________(self) # <<6
  end

  def addItem(item)
    @items.insert(item)
  end
end